RegistrationApp/users.txt

```diff
@@ -0,0 +1,9 @@
+0,Usr2,Avr@123Avr,Ikar,Smit,ad@gmail.com,+380666111111,true,1988-01-01
```

Comment:
loading user information to public is a bad idea))

Reply:
Yep, but it was our split desigion. What do you suggest to do with this?

Reply:
i couldn't suggest you to load users to git